tracee-rules/engine/engine.go

```diff
@@ -5,6 +5,7 @@ import (
 	"log"
 
 	"github.com/aquasecurity/tracee/tracee-rules/types"
+	tracee "github.com/aquasecurity/tracee/tracee/external"
```

Comment:
I'm confused by this import path. To me it looks like the `tracee/external` package is in `github.com/aquasecurity/tracee/tracee-ebpf/tracee/external`, how does this work when the `tracee-ebpf` part is dropped?

Reply:
you are looking at HEAD but go import gets the packages from latest tag. in tandem with the release we would need to updated these paths, and I've opened #497 as a reminder